install/kubernetes/templates/istio-auth/istio-cluster-ca.yaml.tmpl

```diff
@@ -1,23 +1,17 @@
-# istio-system namespace for Istio CA.
-apiVersion: v1
-kind: Namespace
```

Comment:
what creates it then?

Reply:
This is just being moved to a different file.